.coafile

```diff
@@ -0,0 +1,26 @@
+[all]
+ignore = node_modules/**, bower_components/**, dist/**, .eslintrc.js, tmp/**,
+         .git/**
+
+[all.SpaceConsistencyBear]
+files = **.js, **.json, **.yml, **.md, .eslintrc
+bears = SpaceConsistencyBear
+use_spaces = true
+
+[all.JSONFormatBear]
+bears = JSONFormatBear
+files = **.json
+indent_size = 2
+
+[all.ESLintBear]
+bears = ESLintBear
+eslint_config = .eslintrc.js
+files = *.js, app/**.js, config/*.js, tests/**.js
+
+[all.ESLintBear]
```

Comment:
Should be `all.markdown` ?

The section name doesn't have to be the bear name

The reason why it shouldn't be the bear name, because we might add more bear into the section.

The reason why we needed the `SpaceConsistencyBear` to be in a separate section is because it should run on all files and the section name doesn't have to be `SpaceConsistencyBear` if we want it consistent with the rest of coala repos.

Maybe see https://github.com/coala/community/blob/master/.coafile and https://github.com/coala/gci-leaders/blob/master/.coafile for naming.

Reply:
ping @jayvdb

Reply:
(yes, section names shouldnt be bear names as a general rule)

it has been renamed to `all.js`, so this is good now.